examples/v1alpha3/quick-start-example.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: "kubeflow.org/v1alpha3"
+kind: Experiment
+metadata:
+  namespace: kubeflow
+  name: quick-start-example
+spec:
+  parallelTrialCount: 3
+  maxTrialCount: 12
+  maxFailedTrialCount: 3
+  objective:
+    type: maximize
+    # It is a hack to show the functionality of katib, in production we can use 1.
```

Comment:
I didn't get this logic. Can you explain more?

Reply:
If we set it to 1, the experiment will be finished soon (with 1-2 trials).